packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -236,6 +236,11 @@ const AccountWizard = observer(props => {
                         ? getLocation(residence_list, values.tax_residence, 'value')
                         : values.tax_residence;
                 }
+                if (values.address_state) {
+                    values.address_state = values.address_state
+                        ? getLocation(states_list, values.address_state, 'value')
```

Comment:
This code block will be executed only if values.address_state is true. isnt?
why do we need values.address_state check again inside if. block?

Reply:
@amina-deriv we have the same logic for all the above fields as well , I just fallowed the same code style but your right , we can remove all of them. let's move it to another card since it's not in the scope of this card.